feat(fonts)!: local provider unification (#15213)
withastro/astro: .changeset/dull-cities-hang.md

---
'astro': patch
---

**BREAKING CHANGE to the experimental Fonts API only**

Updates how the local provider must be used when using the experimental Fonts API

Previously, there were 2 kinds of font providers: remote and local.

> Font providers are now unified. If you are using the local provider, here is how you can update:

Reviewer: ```suggestion
Font providers are now unified. If you are using the local provider, the process for configuring local fonts must be updated:
```
